modules/Activities/activities_attendance.php

```diff
@@ -41,6 +41,8 @@
     echo __($guid, 'Choose Activity');
     echo '</h2>';
 
+    $highestAction = getHighestGroupedAction($guid, '/modules/Activities/activities_attendance.php', $connection2);
+    print $highestAction;
```

Comment:
Great looking commit! Looks like you may have left a debugging output in here ;) If you push any new changes to your branch it'll automatically update the PR